.circleci/config.yml

```diff
@@ -13,14 +13,20 @@ machine_ubuntu: &machine_ubuntu
 android_machine_ubuntu: &android_machine_ubuntu
   working_directory: ~/circleci-caver-java-android
   machine:
-    image: ubuntu-1604:201903-01
+    image: android:202102-01
```

Comment:
is this image support by circleci?

Reply:
@whoisxx 

Yeap. 
you can check it here "[CircleCI Android machine image preview docs](https://github.com/CircleCI-Public/android-image-preview-docs#no-orb-example)" :)